assets/node-exporter/prometheus-rule.yaml

```diff
@@ -55,10 +55,10 @@ spec:
         description: Filesystem on {{ $labels.device }} at {{ $labels.instance }}
           has only {{ printf "%.2f" $value }}% available space left.
         runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/cluster-monitoring-operator/NodeFilesystemAlmostOutOfSpace.md
-        summary: Filesystem has less than 5% space left.
+        summary: Filesystem has less than 3% space left.
       expr: |
         (
-          node_filesystem_avail_bytes{job="node-exporter",fstype!=""} / node_filesystem_size_bytes{job="node-exporter",fstype!=""} * 100 < 5
+          node_filesystem_avail_bytes{job="node-exporter",fstype!=""} / node_filesystem_size_bytes{job="node-exporter",fstype!=""} * 100 < 3
```

Comment:
hmm the change comes from https://github.com/prometheus/node_exporter/pull/2352 which is supposed to fix the critical/warning thresholds, not sure why we get the opposite result :thinking:

Reply:
Not the opposite result, but exactly what the code says :)
https://github.com/prometheus/node_exporter/blob/7fc5c6045aeb1d615296f6daca1f7a77554d5efb/docs/node-mixin/config.libsonnet#L52-L53

Reply:
https://github.com/prometheus/node_exporter/pull/2373

Reply:
:man_facepalming: the thresholds are swapped in mixin config. Thanks for the [PR](https://github.com/prometheus/node_exporter/pull/2373) :)